certified-connectors/Telstra SMS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,720 @@
+    "description": "Send SMS \u0026 MMS, leveraging Australia\u0027s leading mobile network.",
+    "version": "1.0",
+    "termsOfService": "https://www.telstra.com.au/customer-terms",
+    "contact": {
+      "name": "Telstra Messaging API",
+      "url": "https://dev.telstra.com/apis/messaging-api",
+      "email": "telstradev@team.telstra.com"
+    }
+  },
+  "host": "products.api.telstra.com",
+  "basePath": "/messaging/v3",
+  "schemes": ["https"],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/messages": {
+      "post": {
+        "responses": {
+          "400": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "issue": {
+                        "type": "string",
+                        "description": "issue"
+                      },
+                      "location": {
+                        "type": "string",
+                        "description": "location"
+                      },
+                      "value": {
+                        "type": "string",
+                        "description": "value"
+                      },
+                      "fieldName": {
+                        "type": "string",
+                        "description": "fieldName"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "errors": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "code": {
+                        "type": "string",
+                        "description": "code"
+                      },
+                      "issue": {
+                        "type": "string",
+                        "description": "issue"
+                      },
+                      "suggested_action": {
+                        "type": "string",
+                        "description": "suggested_action"
+                      }
+                    }
+                  },
+                  "description": "errors"
+                }
+              }
+            }
+          },
+          "403": {
+            "description": "default",
```

Comment:
please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc. but not default